### ISSUE TITLE
Hori FC Octa for xb1 not working

### ISSUE THREAD

**Issue by jvook** [o0Zz/sys-con#27]
**Information**
Atmosphere version: 1.7.1
Switch OS version: 18.1.0
Syscon version: v1.2.2 with latest config
Controller: Hori FC Octa Xbox One/S/X (https://stores.horiusa.com/fighting-commander-octa-designed-for-xbox-series-x-s-xbox-one/)

**Description**
I am trying to get my Hori to work.  It used to work on the old distro of sys-con without any configuration, presumably as a Xbox One controller.  When I connect the wire, the xbox button blinks once and there is no further response.  I have tested a buffalo SNES pad and it does work in the sys-con troubleshooter app.

I downloaded the newest config.ini.  That did not help. I then got the USB VID/PID from windows, and made an entry with xboxone driver.  It did not work. (0f0d-0161)

I then went to type this report, and then I saw to look at troubleshooting and logs.  I had done the troubleshooting and it did not help, but when I saw the logs, I saw it kept picking up a device with a different VID/PID (046d-c52f).  I tried adding this to the config.ini but it started making atmosphere crash whenever I connect the xbox controller (screenshot attached).

**Logs**

[log.log](https://github.com/user-attachments/files/16430317/log.log)
[config.zip](https://github.com/user-attachments/files/16430314/config.zip)
[fatal_errors.zip](https://github.com/user-attachments/files/16430377/fatal_errors.zip)

![PXL_20240730_152037937](https://github.com/user-attachments/assets/7f58fac5-eee0-4827-8cb9-2e4b37b4df06)

edit: added the atmosphere log
edit 2: i actually have two of these controllers (two dongles too) and they are both having the issue, so i'm pretty sure it's not a hardware issue.  They both work in windows.  It's an official dock and i tried all the ports.

**Comment by o0Zz** [comment]
Hi,
Thank you for the report.

Could you try this version: 
[sys-con-1.2.2+16-ATMOSPHERE-1.7.x.zip](https://github.com/user-attachments/files/16432729/sys-con-1.2.2%2B16-ATMOSPHERE-1.7.x.zip)

And could provide the logs in Trace mode please -> https://github.com/o0Zz/sys-con/blob/master/doc/Troubleshooting.md#how-to-attach-a-logs-to-a-ticket-

Thanks

**Comment by o0Zz** [comment]
Any update on this ticket ?

**Comment by jvook** [comment]
Thanks for the updated file to try! It didn't seem to help, and still seeing the same issue of blinking one time then turning off.  Here is the trace log.  
[sys-con.zip](https://github.com/user-attachments/files/16464939/sys-con.zip)

**Comment by o0Zz** [comment]
Hi,

Could you please try this version and provide the logs.
(It should not works BUT I will have more logs to debug the problem - I think we will be able to solve this problem).

[sys-con-1.2.2+19-ATMOSPHERE-1.7.x.zip](https://github.com/user-attachments/files/16466402/sys-con-1.2.2%2B19-ATMOSPHERE-1.7.x.zip)

Thanks

**Comment by jvook** [comment]
That would be great, I love these controllers for the stuff I do with my switch! Here is the log for v19 update, same result as you expected.
[log.log](https://github.com/user-attachments/files/16472064/log.log)

**Comment by o0Zz** [comment]
Could you try to add this to your config.ini:

```
[046d-c52f]
driver=xboxone
```

Be carefull the section is probably already present at the end of the file.

**Comment by jvook** [comment]
Okay so I tried with that ID and the controller did register and works in the switch menu.  But it was glitchy, sort of like it was lagging, and was jumping 5+ items at a time and stuff like that.  

Also an item was added to my config for 0f0d-0150 which sounds like a hori controller but not the one I have? It didn't seem to do anything.  The controller only worked if I have the 046d-c52f turned on.

[config.zip](https://github.com/user-attachments/files/16477876/config.zip)
